content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+date: 2019-11-13T11:23:04-06:00
+draft: false
+meta_desc: "Pulumi supports .NET Core languages for Infrastructure as Code"
+meta_image: "meta.png"
+authors:
+    - sophia-parafina
+tags:
+    - ".Net Core"
+    - "C#"
+    - "F#"
+    - "Visual Basic"
+---
+
+We are excited to announce the preview of Pulumi .NET support. Pulumi lets you declare infrastructure using general purpose programming languages. Our goal is to automate applications at all layers and help developers and operators work better together by eliminating the silos that separate them.
+
+Millions of .NET developers can now use their favorite languages and open source ecosystems to build modern, cloud native applications. We've added support for C#, F#, and Visual Basic. Because .NET Core is available on Windows, Linux, and macOS, you have a choice of platforms to use. You can create cloud resources by writing Microsoft .NET Core programs to build and deploy AKS Clusters, Functions, Azure App Services, Virtual Machines, Cosmos DBs, KeyVaults, and much, much more.
```

Comment:
```suggestion
Millions of .NET developers can now use their favorite languages and open source ecosystems to build modern, cloud native applications. We've added support for C#, F#, and Visual Basic .NET. Because .NET Core is available on Windows, Linux, and macOS, you have a choice of platforms to use. You can create cloud resources by writing Microsoft .NET Core programs to build and deploy cloud resources to a wide variety of clouds, including Azure, AWS, GCP and more.  On Azure, you can manage resources like AKS Clusters, Functions, Azure App Services, Virtual Machines, Cosmos DBs, KeyVaults, and much, much more.
```